src/style.css

```diff
@@ -1,3 +1,84 @@
-body {
+html {
+  background-color: #e5e5e5;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+
+  --secondary: #616070;
+  --main-accent: #060b35;
+  --white: #fff;
+  --blue-accent: #00acdc;
+}
+
+body,
+div,
+a,
+h1,
+h2,
+img,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
It is necessary to set the width for the card, but the height will be expanded by the content, because if we do not limit the width, it will be full screen, as it is now